lib/sensors/i2c_mux.hpp

```diff
@@ -10,19 +10,24 @@
 
 namespace hyped::sensors {
 
-static constexpr std::uint8_t kDefaultMuxAddress = 0x70;
-static constexpr std::uint8_t kMaxNumChannels    = 8;
-static constexpr core::Float kFailureThreshold
-  = 0.25;  // TODOLater: finalize this value with Electronics
+constexpr std::uint8_t kDefaultMuxAddress = 0x70;
+constexpr std::uint8_t kMaxNumChannels    = 8;
+// perentage of sensors that can be unusable before the mux is considered unusable
```

Comment:
Would we not want more specific fail states (i.e. "all temperature sensors failing" instead of just a proportion of all sensors)?

Reply:
this is what electronics wanted, and double confirmed (i personally dont agree with this either but...)

Reply:
This is very upsetting news. Can we do more specific error handling anyways and then just given them that value at the end?